src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -10440,6 +10440,13 @@ private static void payActionCost(final String responseText) {
           skillSuccess = true;
         }
         break;
+
+      case SkillPool.LAY_AN_EGG:
+        if (responseText.contains("shudders and lays an egg") || skillSuccess) {
```

Comment:
Where was skillSuccess set prior to this being executed?

Reply:
up top, if you run with haiku / anapests / in the deep machine tunnels

Reply:
Thanks.  There's got to be a better way to organize 10K lines of code for human readability.